src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -21,6 +21,13 @@ public override BoundNode VisitConversion(BoundConversion node)
                 return RewriteInterpolatedStringConversion(node);
             }
 
+            if (node.ConversionKind is ConversionKind.SwitchExpression or ConversionKind.ConditionalExpression)
```

Comment:
I wasn't sure why it's necessary to skip these conversions in LocalRewriter. Could you please clarify?

Reply:
These don't get emitted. These nodes are newly added in initial binding, they didn't exist previously.

---
In reply to: [594569914](https://github.com/dotnet/roslyn/pull/51791#discussion_r594569914) [](ancestors = 594569914)